plasma_framework/python_tests/testlang/testlang.py

```diff
@@ -292,16 +292,22 @@ def start_fee_exit(self, operator, amount, token=NULL_ADDRESS, bond=None):
                                                **{'value': bond, 'from': operator.address, 'gas': 1_000_000})
         return fee_exit_id, tx_hash
 
-    def process_exits(self, vault_id, token, exit_id, count=1, **kwargs):
+    def process_exits(self, token, exit_id, count=1, vault_id=None, **kwargs):
         """Finalizes exits that have completed the exit period.
 
         Args:
-            vault_id (int): Id of the vault that funds the exit
             token (address): Address of the token to be processed.
             exit_id (int): Identifier of an exit (optional, pass 0 to ignore the check)
             count (int): Maximum number of exits to be processed.
+            vault_id (int): Id of the vault that funds the exit
         """
 
+        if not vault_id:
```

Comment:
minor: I vaguely remember reading sth that it is preferred to write `if vault_id is None` instead of `if not vault_id` for better readability somewhere.

Reply:
sure! I somehow forgot about it